Makefile

```diff
@@ -151,8 +151,8 @@ local-%:  ## Builds the specified target defined in the Dockerfile using the loc
 	    echo $$platform; \
 	    directory="$${platform//\//_}"; \
 	    if [[ -d "$$DEST/$$directory" ]]; then \
-	      mv "$$DEST/$$directory/"* $$DEST; \
```

Comment:
can we instead force a single platform output for `make generate`? copying hundreds of megabytes doesn't sound like fun (but I guess it solves the issue when output is a directory?)

Reply:
changed the PR to do that instead.